src/governance/classes.cpp

```diff
@@ -385,6 +385,13 @@ void CSuperblockManager::ExecuteBestSuperblock(CGovernanceManager& governanceMan
         // All checks are done in CSuperblock::IsValid via IsBlockValueValid and IsBlockPayeeValid,
         // tip wouldn't be updated if anything was wrong. Mark this trigger as executed.
         pSuperblock->SetExecuted();
+        // Remove all used Governance objects: approved Proposal and executed trigger
+        auto proposals = pSuperblock->GetProposalHashes();
+        for (const auto& h : proposals) {
+            governanceManager.DeleteGovernanceObject(h);
+        }
+        auto trigger = pSuperblock->GetGovernanceObject(governanceManager);
+        governanceManager.DeleteGovernanceObject(trigger->GetHash());
```

Comment:
Well, I was still seeing the trigger and the paid proposals after the SB was executed when calling `gobject list`.
It confused me but maybe this is business-wide acceptable.
Reverting the change.

Reply:
That's by design:
1. some proposals are paid in multiple cycles
2. we keep executed triggers in memory for some time to prevent sb reorgs

We handle expired proposals and triggers already, there should be no need to implement any additional logic for them I think.